Auth: Add feature flag to move token rotation to client (#65060)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

| `prometheusMetricEncyclopedia`     | Replaces the Prometheus query builder metric select option with a paginated and filterable component                                                                         |
| `timeSeriesTable`                  | Enable time series table transformer & sparkline cell type                                                                                                                   |
| `influxdbBackendMigration`         | Query InfluxDB InfluxQL without the proxy                                                                                                                                    |

> | `clientTokenRotation`              | Replaces the current in-request token rotation and forces the rotation to happen from client                                                                                 |

Reviewer: ```suggestion
| `clientTokenRotation`              | Replaces the current in-request token rotation so that the client initiates the rotation                                                                                 |
```

Author: I updated the description. This text is auto generated so had to update the text in another place to make it correct
